src/uni_modules/wot-ui/components/wd-button/index.scss

```
@@ -314,6 +368,10 @@ $button-variants: ("text": ("hide-border": true),
             background: transparent;
           }
 
+          @else if map.get($variant-conf, "soft-bg") {
+            background: map.get($type-conf, "soft-bg");
+          }
```

Stylelint (17.6.0)

[error] Unexpected newline after "}" of `@else` statement (scss/at-else-closing-brace-newline-after)

(scss/at-else-closing-brace-newline-after)